src/main/java/org/apache/commons/codec/binary/Base64.java

```diff
@@ -265,6 +365,11 @@ public static byte[] decodeBase64(final byte[] base64Data) {
      * Decodes a Base64 String into octets.
      * <p>
      * <strong>Note:</strong> this method seamlessly handles data encoded in URL-safe or normal mode.
+     * For enforcing verification against strict standard Base64 or Base64 URL Safe tables,
+     * please use {@code #decodeBase64Standard} or {@code decodeBase64Url} methods respectively.
```

Comment:
Use `@link` where you can instead of `@code`. Saying `{@code #...` doesn't mean anything in Javadoc, only in `{@link #...`.